readme.md

```diff
@@ -22,3 +22,22 @@ serve({
   404: () => new Response("not found"),
 });
 ```
+
+### `jsx()`
+
+Render JSX components to produce Response with `text/html` content type.
+
+```js
+import { jsx, serve } from "https://deno.land/x/sift@0.1.7/mod.ts";
```

Comment:
```suggestion
import { jsx, serve, h } from "https://deno.land/x/sift@0.1.7/mod.ts";
```

Sorry, I didn't mention it before but we can import `h` from sift. Sift uses a modified version of preact with types that are supported on Deno.